### ISSUE TITLE
Spike: AXE DevTools Enterprise integration for automated storefront testing

### ISSUE THREAD

**Issue by o-hook** [porsche-design-system/porsche-design-system#461]
**Scope**
Integrate automated accessibility testing into storefront to evaluate the ease of integration, use and value compared to other solutions like Lighthouse or Storybook AXE integration. Technically it should be based on AXE DevTools Enterprise HTML which is part of the Deque AXE Tools Suite.

**Out of scope**
- Performance, focus on accessibility reports
- No CI/CD integration relevant for evaluation

**Notes**  
- [AXE DevTools HTML](https://axe-devtools-html-docs.deque.com/reference/home.html) is an API based on AXE Core and comes with prebuild configurations for various testing environments.
- [AXE Core](https://github.com/dequelabs/axe-core) is an open source accessibility testing engine to integrate the tests in your environment.
- Deque Sample projects can be downloaded here: https://sftp.dequecloud.com/?u=pmqZbbHJ4Z&p=UvyVVTgNju
- AXE DevTools Registry (needs permissions): @axe-devtools:registry=https://agora.dequecloud.com/artifactory/api/npm/devtools-npm/
- Onboarding sessionsfrom Deque will be given

Acceptance Criteria
---
- [x] Create an automatic a11y testing with AXE DevTools for storefront

Decision
---
We're using `@axe-core/puppeteer` over `@axe-devtools/puppeteer` since result wise they are identical.
The only potential benefit of `@axe-devtools/reporter` is the possibility to create html reports.
